evalml/model_understanding/graphs.py

```diff
@@ -552,7 +552,7 @@ def _raise_value_error_if_mostly_one_value(df, percentile):
 
 
 def partial_dependence(
-    pipeline, X, features, percentiles=(0.05, 0.95), grid_resolution=100
+    pipeline, X, features, percentiles=(0.05, 0.95), grid_resolution=100, kind="average"
```

Comment:
We could potentially make `kind="both"` by default, but it would throw an error when doing a two-way PD. If we changed the behavior to be a silent change or a warning, I would be okay with doing this as a default